src/vcSettings.h

```diff
@@ -339,4 +359,8 @@ udResult vcSettings_RegisterAssetFileHandler();
 // Various settings helpers
 udResult vcSettings_UpdateLanguageOptions(vcSettings *pSettings);
 
+void vcSettings_UpdateServerFlag(vcSettings *pSettings);
+
+const char * vcSettings_GetMapTileTypeName(vcSettings *pSettings);
```

Comment:
These functions are perfect candidates to be inlined

Reply:
All right.

Reply:
The cost of inlining here is too high, so you need to change where g_vcMapTileTypeName and g_vcMapTileServerName are declared